### DISCUSSION TITLE
Feature request : scramble preview color customization

### DISCUSSION THREAD

**Discussion by deafgeek** [CubeLabsNZ/CubeTime#64] [Ideas]
I would like to suggest a feature request for the scramble preview on screen. A way to customize the colors used for the faces/pieces.  I do not like the green it too bright and having green next to yellow make it very tricky for me to see it clearly. 

  It could be an option in appearance like cube color picker then have the colors or sides have it own entry then touching on one should open like maybe a couple of presets and a color picker where you can pick from a square with the different colors or a text box to enter the color code like _____ and for example entering #0B610B should get a forest green and could have a couple of online links to a color picker site one example site is  https://html-color-codes.info/ .

**Comment by xbjfk** [comment]
Locked in favor of #69
